CMakeLists.txt

```diff
@@ -119,7 +119,7 @@ add_executable(version_management EXCLUDE_FROM_ALL examples/by_runtime_api_modul
 add_executable(error_handling EXCLUDE_FROM_ALL examples/by_runtime_api_module/error_handling.cu)
 add_executable(device_management EXCLUDE_FROM_ALL examples/by_runtime_api_module/device_management.cpp)
 add_executable(execution_control EXCLUDE_FROM_ALL examples/by_runtime_api_module/execution_control.cu)
-target_compile_options(execution_control PRIVATE -rdc true)
+#target_compile_options(execution_control PRIVATE -rdc true)
```

Comment:
IIRC, this is necessary on non-Windows machines, I can't just remove it.

Reply:
Ok - I'll make the change conditional then.